Realm/RLMCollection.mm

```diff
@@ -149,6 +150,17 @@ - (NSUInteger)countByEnumeratingWithState:(NSFastEnumerationState *)state
         return results;
     }
 
+    for (RLMProperty *prop in info->rlmObjectSchema.swiftGenericProperties) {
+        if ([prop.name isEqual:key] && prop.swiftIvar && !prop.optional) {
```

Comment:
Because Swift optional properties can be used accessor reused version. I'd like to use creating a new accessor object for each value version only for `List` and `LinkingObjects`. It is two times slower than the accessor reused version.